ca/ca.go

```diff
@@ -288,6 +298,24 @@ func (ca *CA) Run() error {
 	var wg sync.WaitGroup
 	errs := make(chan error, 1)
 
+	if !ca.opts.quiet {
+		authorityInfo := ca.auth.GetInfo()
+		log.Printf("Address: %s", ca.config.Address)
+		for _, crt := range authorityInfo.RootX509Certs {
+			sum := sha256.Sum256(crt.Raw)
+			log.Printf("X.509 Root Fingerprint: %s", hex.EncodeToString(sum[:]))
```

Comment:
```suggestion
			log.Printf("X.509 Root Fingerprint: %s", x509util.Fingerprint(crt))
```
Needs an import of "go.step.sm/crypto/x509util"